### PR TITLE
Draft: Fix boolean inserts for postgres

### PR DESCRIPTION
**Pull request by filmor**

### CODE REVIEW

File: adm_program/installation/install_steps/start_installation.php [hunk 1]
```diff
@@ -150,33 +150,33 @@
...
 fa-venus|SYS_FEMALE
-fa-mars-stroke-v|SYS_DIVERSE\', 0, 0, 0, 0, 11, '.$gCurrentUserId.', \''. DATETIME_NOW.'\')
-             , ('.$categoryIdMasterData.', \'' . Uuid::uuid4() . '\', \'EMAIL\',        \'EMAIL\',      \'SYS_EMAIL\',     NULL, 0, NULL, 1, 0, 2, 1, 12, '.$gCurrentUserId.', \''. DATETIME_NOW.'\')
-             , ('.$categoryIdMasterData.', \'' . Uuid::uuid4() . '\', \'URL\',          \'WEBSITE\',    \'SYS_WEBSITE\',   NULL, 0, NULL, 0, 0, 0, 0, 13, '.$gCurrentUserId.', \''. DATETIME_NOW.'\')
-             , ('.$categoryIdAddidionalData.', \'' . Uuid::uuid4() . '\', \'CHECKBOX\', \'DATA_PROTECTION_PERMISSION\', \'SYS_DATA_PROTECTION_PERMISSION\', \''.$gL10n->get('SYS_DATA_PROTECTION_PERMISSION_DESC').'\', 1, NULL, 0, 0, 2, 0, 14, '.$gCurrentUserId.', \''. DATETIME_NOW.'\')';
+fa-mars-stroke-v|SYS_DIVERSE\', false, false, false, false, 11, '.$gCurrentUserId.', \''. DATETIME_NOW.'\')
+             , ('.$categoryIdMasterData.', \'' . Uuid::uuid4() . '\', \'EMAIL\',        \'EMAIL\',      \'SYS_EMAIL\',     NULL, false, NULL, true, false, true, true, 12, '.$gCurrentUserId.', \''. DATETIME_NOW.'\')
```
Comment (filmor) [inline adm_program/installation/install_steps/start_installation.php:167]: The 2 is incorrectly converted, that column is apparently a number.

File: adm_program/installation/install_steps/start_installation.php [hunk 2]
```diff
@@ -150,33 +150,33 @@
...
+             , ('.$categoryIdMasterData.', \'' . Uuid::uuid4() . '\', \'URL\',          \'WEBSITE\',    \'SYS_WEBSITE\',   NULL, false, NULL, false, false, false, false, 13, '.$gCurrentUserId.', \''. DATETIME_NOW.'\')
+             , ('.$categoryIdAddidionalData.', \'' . Uuid::uuid4() . '\', \'CHECKBOX\', \'DATA_PROTECTION_PERMISSION\', \'SYS_DATA_PROTECTION_PERMISSION\', \''.$gL10n->get('SYS_DATA_PROTECTION_PERMISSION_DESC').'\', true, NULL, false, false, true, false, 14, '.$gCurrentUserId.', \''. DATETIME_NOW.'\')';
```
Comment (filmor) [inline adm_program/installation/install_steps/start_installation.php:169]: Same as above.

File: adm_program/installation/install_steps/start_installation.php [hunk 3]
```diff
@@ -150,33 +150,33 @@
...
-             , ('.$categoryIdMasterData.', \'' . Uuid::uuid4() . '\', \'PHONE\',        \'PHONE\',      \'SYS_PHONE\',     NULL, 0, NULL, 0, 0, 0, 0, 7,  '.$gCurrentUserId.', \''. DATETIME_NOW.'\')
-             , ('.$categoryIdMasterData.', \'' . Uuid::uuid4() . '\', \'PHONE\',        \'MOBILE\',     \'SYS_MOBILE\',    NULL, 0, NULL, 0, 0, 0, 0, 8,  '.$gCurrentUserId.', \''. DATETIME_NOW.'\')
-             , ('.$categoryIdMasterData.', \'' . Uuid::uuid4() . '\', \'PHONE\',        \'FAX\',        \'SYS_FAX\',       NULL, 0, NULL, 0, 0, 0, 0, 9,  '.$gCurrentUserId.', \''. DATETIME_NOW.'\')
-             , ('.$categoryIdMasterData.', \'' . Uuid::uuid4() . '\', \'DATE\',         \'BIRTHDAY\',   \'SYS_BIRTHDAY\',  NULL, 0, NULL, 0, 0, 0, 0, 10, '.$gCurrentUserId.', \''. DATETIME_NOW.'\')
-             , ('.$categoryIdMasterData.', \'' . Uuid::uuid4() . '\', \'RADIO_BUTTON\', \'GENDER\',     \'SYS_GENDER\',    NULL, 0, \'fa-mars|SYS_MALE
+               (usf_cat_id, usf_uuid, usf_type, usf_name_intern, usf_name, usf_description, usf_description_inline, usf_value_list, usf_system, usf_disabled, usf_mandatory, usf_registration, usf_sequence, usf_usr_id_create, usf_timestamp_create)
```
Comment (filmor) [inline adm_program/installation/install_steps/start_installation.php:153]: Not sure when exactly this one changed, but need to revert.

**Comment by Fasse** [comment]
Thanks for the hints. I understand the problems and will fix them.

**Comment by Fasse** [comment]
Problems should be fixed with #1327

**Pull request closed without merging**